_sass/custom/custom.scss

```diff
@@ -266,18 +268,23 @@ $text-color: #111111;
 // DVLS new styles
 //
 .nav-list {
-    list-style: circle url('/assets/images/nav-marker-inactive.svg') inside;
+  list-style: circle url("/assets/images/nav-marker-inactive.svg") inside;
 
     .nav-list-item {
-	.nav-list-link{
-	    display: inline;
-	}
-	&.active {
-	    list-style: disc url('/assets/images/nav-marker-active.svg') inside;
-	}
+
+	    &.active {
+	      list-style: disc url("/assets/images/nav-marker-active.svg") inside;
+
+	    }
     }
 }
 
+.nav-list-item {
+  .nav-list-link {
+    display: inline;
```

Comment:
This might not be working, I am seeing this in [preview](https://deploy-preview-526--learncircuitverse.netlify.app/) and `nav-list-link` is not showing as expected.

Reply:
> This might not be working, I am seeing this in [preview](https://deploy-preview-526--learncircuitverse.netlify.app/) and `nav-list-link` is not showing as expected.

Sorted, but then, hound bot is showing issue for selector depth

Reply:
Not all hound violations need to be fixed, as it may make code more complex. Add comment to such hound issues and move on